### DISCUSSION TITLE
Multiple classes are mapped to 'Document' name

### DISCUSSION THREAD

**Discussion by BerndDasBrot-1** [LibrePDF/OpenPDF#827] [Q&A]
I got this message: 
'Warning: Multiple classes are mapped to 'Document' name. Conflicting classes: [class com.lowagie.text.Document, interface org.w3c.dom.Document]' at a spring boot application. Any idea what to do. 
Best regards, Bernd

**Comment by asturio** [comment]
Quite a long time ago.
What is not clear, is where are you getting this messages.
If you are using com.lowagie.text.Document and org.w3c.dom.Document in the same class you may only import one of them. The other one you must use the fullname. It's just how Java works.
